src/app/zap-templates/zcl/data-model/chip/pressure-measurement-cluster.xml

```diff
@@ -25,50 +31,41 @@ limitations under the License.
     <client tick="false" init="false">true</client>
     <server tick="false" tickFrequency="half" init="false">true</server>
     <globalAttribute side="either" code="0xFFFD" value="3"/>
-
     <features>
       <feature bit="0" code="EXT" name="Extended" summary="Extended range and resolution">
         <optionalConform/>
       </feature>
     </features>
-    
-    <attribute side="server" code="0x0000" name="MeasuredValue" define="PRESSURE_MEASURED_VALUE" type="int16s" reportable="true" isNullable="true">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="0x0001" name="MinMeasuredValue" define="PRESSURE_MIN_MEASURED_VALUE" type="int16s" isNullable="true">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="0x0002" name="MaxMeasuredValue" define="PRESSURE_MAX_MEASURED_VALUE" type="int16s" isNullable="true">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="0x0003" name="Tolerance" define="PRESSURE_TOLERANCE" type="int16u" min="0x0000" max="0x0800" optional="true" default="0">
+    <attribute side="server" code="0x0000" name="MeasuredValue" define="PRESSURE_MEASURED_VALUE" type="int16s" isNullable="true"/>
+    <attribute side="server" code="0x0001" name="MinMeasuredValue" define="PRESSURE_MIN_MEASURED_VALUE" type="int16s" isNullable="true" min="-32767" max="32766"/>
+    <attribute side="server" code="0x0002" name="MaxMeasuredValue" define="PRESSURE_MAX_MEASURED_VALUE" type="int16s" isNullable="true" min="-32766"/>
+    <attribute side="server" code="0x0003" name="Tolerance" define="PRESSURE_TOLERANCE" type="int16u" max="0x0800" optional="true" default="0">
       <optionalConform/>
     </attribute>
     <attribute side="server" code="0x0010" name="ScaledValue" define="PRESSURE_SCALED_VALUE" type="int16s" optional="true" default="0" isNullable="true">
       <mandatoryConform>
         <feature name="EXT"/>
       </mandatoryConform>
     </attribute>
-    <attribute side="server" code="0x0011" name="MinScaledValue" define="PRESSURE_MIN_SCALED_VALUE" type="int16s" optional="true" default="0" isNullable="true">
+    <attribute side="server" code="0x0011" name="MinScaledValue" define="PRESSURE_MIN_SCALED_VALUE" type="int16s" optional="true" default="0" isNullable="true" min="-32767" max="32766">
       <mandatoryConform>
         <feature name="EXT"/>
       </mandatoryConform>
     </attribute>
-    <attribute side="server" code="0x0012" name="MaxScaledValue" define="PRESSURE_MAX_SCALED_VALUE" type="int16s" optional="true" default="0" isNullable="true">
+    <attribute side="server" code="0x0012" name="MaxScaledValue" define="PRESSURE_MAX_SCALED_VALUE" type="int16s" optional="true" default="0" isNullable="true" min="-32766">
```

Comment:
This seems to be a scraping bug.  In the spec it's `(MinScaledValue + 1) to 32767`

Reply:
This is a little weird, I guess that the max (32767) is omitted since it matches the default value for int16. For the minimum I assume that it would take the minimum possible value of MinScaledValue (-32767 since it is nullable) and then add one (+1), so the minimum for MaxScaledValue will be -32767 + 1, thus -32766. But this would be the whole possible range for that attribute.